docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -892,6 +1011,111 @@ See the <<si-sb-headers, Service Bus message headers>> for the basic message hea
 
 NOTE: When setting the partiton key, the priority of message header is higher than Spring Cloud Stream property. So `spring.cloud.stream.bindings.<binding-name>.producer.partition-key-expression` will take effect only when none of the headers of `ServiceBusMessageHeaders#SESSION_ID`, `ServiceBusMessageHeaders#PARTITION_KEY`, `AzureHeaders#PARTITION_KEY` is configured.
 
+===== Multi Binder Support
+This sample takes connection string as example. Credentials of service priciple and MSI are also supported, users can set related properties in each binder's environment settings.
```

Comment:
same here: Connection to mutiple Service Bus namespaces is also supported by using multi binders.